lib/sort_by_length.rb

```diff
@@ -1,7 +1,26 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2), quadratic, because there are two while loops in the insertion sort function that will both iterate n times.
+# Space complexity: O(n), linear, because the new array that is being stored will increase depending on the size of the data (my_sentence) being passed through.
```

Comment:
Correct since you're doing .split the space complexity is right, and insertion sort his this time complexity.